src/pages/program/[slug]/index.tsx

```diff
@@ -159,41 +115,18 @@ const ReadOnlyHome = ({
 
 //TODO: Change type of this to not any
 const ProgramPage: PageGetProgramBySlugComp & Page = (props: any) => {
-  const authorizationLevel = useAuthorizationLevel();
-
   const program = props.data?.getProgramBySlug;
 
-  switch (authorizationLevel) {
-    case AuthorizationLevel.Admin:
-    case AuthorizationLevel.Mentor:
-    case AuthorizationLevel.Mentee:
-      LocalStorage.set("cachedProgramSlug", program.slug);
-      break;
-    default:
-      break;
-  }
-
   if (!program) {
     return <ErrorScreen type={ErrorScreenType.PageNotFound} />;
   }
 
-  const getProgramPage = () => {
-    switch (authorizationLevel) {
-      case AuthorizationLevel.Admin:
-        return <AdminHome {...program} />;
-      case AuthorizationLevel.Mentee:
-      case AuthorizationLevel.Mentor:
-        return <ReadOnlyHome {...program} inProgram={true} />;
-      default:
-        return (
-          <>
-            <SignedInAsIndicator />
-            <ReadOnlyHome {...program} />
-          </>
-        );
-    }
-  };
-  return <PageContainer>{getProgramPage()}</PageContainer>;
+  return (
+    <PageContainer>
+      <SignedInAsIndicator />
+      <ReadOnlyHome {...program} />
+    </PageContainer>
+  );
 };
 
 export default ProgramPage;
```

Comment:
If you log in / sign up through the apply to _____ into an account that is already in the program, you're allowed to apply again through these pages => if you click apply and reapply to the program it throws a dirty error

1.  Log in through modal
![image](https://user-images.githubusercontent.com/20727715/127485649-a87a99f2-b520-41be-a4de-f418fdaa86c4.png)
2. Click apply to _____
![image](https://user-images.githubusercontent.com/20727715/127485714-40d13261-4c05-4cc5-96e9-f8811aa4de4c.png)
3. Re-apply for the program
![image](https://user-images.githubusercontent.com/20727715/127485778-5028bef3-15dd-498f-b8c9-1b872b722997.png)

Think we should make it impossible for anybody already in the program to access the non-profile route route. It should not be possible for anybody who is logged in and in a program to have access to the non-profile route or we need to take out the ability for them to apply before applying.

Reply:
Or if this is intended because of the ability to apply for multiple profiles and we're just waiting on backend changes, I take back my comment :' )